libs/js/banglejs/Bangle_showLauncher.js

```diff
@@ -1,12 +1,24 @@
 (function() {
-  var l = require("Storage").list(/\.info$/).map(file => {
-    var app = require("Storage").readJSON(file,1);
-    if (app && app.type == "launch") return app;
-  }).find(x=>x);
-  if (l) {
+  var launcherApp=(require("Storage").readJSON("setting.json",1)||{}).launcher;
+  if (require("Storage").read(launcherApp)==undefined) {
+    launcherApp = require("Storage").list(/\.info$/)
+      .map(file => {
+        const app = require("Storage").readJSON(file,1);
+        if (app && app.type == "launch") {
+          return app;
+        }
+      })
+      .filter(x=>x)
+      .sort((a, b) => a.sortorder - b.sortorder)[0].src;
+  }
```

Comment:
Actually, how about we update the settings here with the launcher we found?

Because as-is, this is going to slow down everyone's device *until* they manually set the launcher in settings. And then if they change launcher via the app loader and forget to change the settings, it'll silently end up slow again